.travis.yml

```diff
@@ -68,6 +68,7 @@ jobs:
         - if [[ "$WITH_COVERAGE" == "true" ]]; then xdebug-enable; fi
         - if [[ "$WITH_COVERAGE" == "true" ]]; then vendor/bin/phpunit --testsuite unit --coverage-clover build/logs/clover.xml; else vendor/bin/phpunit --testsuite unit; fi
         - if [[ "$WITH_COVERAGE" == "true" ]]; then xdebug-disable; fi
+        - vendor/bin/behat
```

Comment:
Can we add this command to the `Makefile` as well?
Something like
```make
behat: composer test-env
	vendor/bin/behat
```

And then also add it to the `make test` command